app/views/questions/show.html.haml

```diff
@@ -1,16 +1,19 @@
 %p#notice= notice
-
 %p
   %b Created by:
   = get_user(@question.user_id).username
   %h2 Question:
   %p
     = @question.content
   %h2 Answers:
-  - @question.answers.each.with_index do |a,i|
-    - if a.content != nil
+  = form_tag submit_question_url do
+    = hidden_field_tag 'question', @question.id
+    - @question.answers.each.with_index do |a,i|
       = label_tag a.content
-      = radio_button :answer, "value", id: i
+      = radio_button_tag :answer, a.id
+    %br
+    = submit_tag 'submit'
```

Comment:
I will open a separate ticket for this. I wanted to get this in, so I put this off for now.